frontend/src/screens/researcher/studies/create/forms/review-study.tsx

```diff
@@ -172,6 +173,8 @@ export const SubmitStudyModal: FC<{
     const api = useApi()
     const [submitted, setSubmitted] = useState(false)
     const [submitting, setSubmitting] = useState(false)
+    const [, , removeStudyProgressStep] = useLocalstorageState<StudyStep>(`study-progress-${study.id}`)
+    const [, , removeMaxStep] = useLocalstorageState<StudyStep>(`study-max-progress-${study.id}`)
     if (submitted) {
```

Comment:
interesting, I didn't know you could leave the var blank, I always used `_` as placeholders.  I learned something new here ❤️

Reply:
I only figured this out cause my built in IDE linter would complain about the `_` and it drove me nuts 😆